pool/payment_test.go

```diff
@@ -144,3 +144,63 @@ func testArchivedPaymentsFiltering(t *testing.T, db *bolt.DB) {
 		t.Fatalf("emptyBucket error: %v", err)
 	}
 }
+
+func testAccountPayments(t *testing.T, db *bolt.DB) {
+	count := uint32(2)
+	amt, _ := dcrutil.NewAmount(5)
+
+	// Create pending payments for account X.
+	pbx := makePaymentBundle(xID, count, amt)
+	for _, pmt := range pbx.Payments {
+		err := pmt.Create(db)
+		if err != nil {
+			t.Fatal(err)
+		}
+	}
+
+	time.Sleep(time.Millisecond * 10)
```

Comment:
I dont see why this sleep is necessary. As far as I can tell, everything above, including the DB call in `pmt.Create`, is synchronous

Reply:
you're right, don't need to sleep there.